tests/lib/caseholder.js

```diff
@@ -0,0 +1,60 @@
+// Caseholder for easy reuse in mail tests.
+"use strict";
+
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.caseholder");
+
+var jqUnit = require("jqUnit");
+var fs     = require("fs");
+
+gpii.test.mail.caseholder.verifyMessage = function (that, session, expected, messageFile) {
+
+    jqUnit.assertEquals("The sender should be correct", expected.from, session.envelope.mailFrom.address);
+    jqUnit.assertEquals("The recipient should be correct", expected.to, session.envelope.rcptTo[0].address);
+
+    if (messageFile) {
+        // Confirm that the test content exists and is correct
+        var messageBody = fs.readFileSync(messageFile, "utf8");
+        jqUnit.assertTrue("There should be a message ID", messageBody.indexOf("Message-Id") !== -1);
+        jqUnit.assertTrue("The subject data should be in the message.", messageBody.indexOf(expected.subject) !== -1);
+        jqUnit.assertTrue("The message text should be in the message.", messageBody.indexOf(expected.text) !== -1);
+    }
+};
+
+// Copied from a similar utility grade in `gpii.express`.
+gpii.test.mail.caseholder.addRequiredSequences = function (sequenceStart, rawTests) {
```

Comment:
Sorry, the comment appeared to be about the previous function.  I would rather not introduce gpii-express as a dev dependency just to pick this up, but I can do so.